core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/workspace/config/MachineConfig.java

```diff
@@ -28,6 +28,13 @@
    */
   String MEMORY_LIMIT_ATTRIBUTE = "memoryLimitBytes";
 
+  /**
+   * Name of the attribute from {@link #getAttributes()} which if present defines memory limit of
```

Comment:
Javadocs of this attribute is the same as javadocs of `memoryLimitBytes`. Is it on purpose?